src/network/networkd-link-bus.c

```diff
@@ -854,7 +854,7 @@ int link_send_changed_strv(Link *link, char **properties) {
         assert(link->manager);
         assert(properties);
 
-        if (!link->manager->bus)
+        if (!link->manager->bus || sd_bus_is_ready(link->manager->bus) <= 0)
```

Comment:
Shouldn't we make sd_bus_is_ready() silently return false on NULL? All callers have the same != NULL check first...

Reply:
https://github.com/systemd/systemd/pull/19250